docs/technical-documentation/dev-process/Enumeration Handling.md

```diff
@@ -0,0 +1,40 @@
+## Enumeration
+
```

Comment:
this entire file seems quite policy hub specific

Reply:
removed